hack/go-linter.sh

```diff
@@ -13,6 +13,7 @@ golangci-lint run --disable-all \
     --enable=ineffassign \
     --enable=goconst \
     --enable=goimports \
+    --enable=dupl \
 
 ##todo(camilamacedo86): The following checks requires fixes in the code
 # --enable=golint
```

Comment:
please, could you remove from the commented lines as well?

Reply:
Yes sure.